### PR TITLE
fix before migration (#56)

### PR DESCRIPTION
**Pull request by debam-das**
* fix for xref CP data
* add env option for CP url

### CODE REVIEW

File: suites/test_bulk_operations.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def modules_bulk_edit_metadata(self):
         utilities.page_reload(self.driver)
         search_beta_page.select_repo(self.driver, fixture.repo_name)
         search_beta_page.search_titles(self.driver, constants.bulk_operations_modules)
-        poll(lambda: len(utilities.find_elements_by_css_selector(self.driver, locators.MODULES_LIST_CSS))== 4,
+        poll(lambda: len(utilities.find_elements_by_css_selector(self.driver, locators.MODULES_LIST_CSS))== 5,
```
Comment (noopurmath3) [inline suites/test_bulk_operations.py:31]: I have added a comment here https://gitlab.cee.redhat.com/ccs-tools/uploader-sample-repo-pantheon2/-/merge_requests/11/diffs

The search result here should return 4 values only and all the changes you have made to change to count from 4 to 5 and 3 to 4 should not be required

File: helpers/constants.py [hunk 2]
```diff
@@ -141,7 +141,7 @@
 xref_validation_module_name = "at-uploader | This doc contains a list of refs"
 # xref validation constants in CP
 Xref_linkText1 = "at-uploader | Assembly Publish test"
-Xref_linkText2 = "Content Test Module | Logging in to the web console using Kerberos authentication (Image present)"
+Xref_linkText2 = "at-uploader | Module bulk operations 4"
```
Comment (noopurmath3) [inline helpers/constants.py:144]: Please update this as per changes suggested here https://gitlab.cee.redhat.com/ccs-tools/uploader-sample-repo-pantheon2/-/merge_requests/11/diffs

**Review by noopurmath3 — CHANGES_REQUESTED** [review]

File: suites/test_bulk_operations.py (hunk 1 above)
Comment (debam-das) [reply suites/test_bulk_operations.py:31]: as uploader repo content is changed reverting all changes.

File: helpers/constants.py (hunk 2 above)
Comment (debam-das) [reply helpers/constants.py:144]: Changed uploader repo's hyperlink "Different module included in the same assembly" to neglect  "bulk operations" substring.

**Review by noopurmath3 — APPROVED** [review]

**Pull request merged**